pom.xml

```diff
@@ -96,6 +97,11 @@
             <artifactId>bullet-core</artifactId>
             <version>${bullet.core.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.yahoo.bullet</groupId>
+            <artifactId>bullet-dsl</artifactId>
```

Comment:
This will pull in kafka, pulsar and whatever else clients in the future. Can we use the DSL by having the jar at runtime and make it provided here? We need to try that out in storm.

Reply:
Ok. I've tried this out. A couple problems but works for the most part. 

1) I get a JNI error from the bullet-storm fat jar. I fixed this by making bullet-dsl "provided" and giving the jar to storm separately.
2) There's a pulsar dependency in BulletDSLConfig. Oops.
3) BulletConnector isn't serializable. Oops.

Reply:
Let's fix bullet-dsl before merging this then